### ISSUE TITLE
Mastodon docker install tries to use ws instead of wss

### ISSUE THREAD

**Issue by Manishearth** [tootsuite/mastodon#6145]
On my brand new instance setup after logging in I get the following errors:

![image](https://user-images.githubusercontent.com/1617736/34467264-fc2af15c-ef11-11e7-82d9-b813982d4660.png)

As well as one error that says

```
Mixed Content: The page at 'https://oxidized.systems/web/getting-started' was loaded over HTTPS, but attempted to connect to the insecure WebSocket endpoint 'ws://oxidized.systems/api/v1/streaming/?stream=user&access_token=TOKEN'. This request has been blocked; this endpoint must be available over WSS.
```

I've done the docker and NGINX setup, it's unclear why the app is trying the insecure websocket connection.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by Manishearth** [comment]
Ah, the issue is that mastodon still needs `LOCAL_HTTPS=true` set even if it [says it doesn't](https://github.com/tootsuite/mastodon/blob/master/.env.production.sample#L18)

**Comment by ykzts** [comment]
That change is not included in v2.1.0 ([`.env.production.sample`](https://github.com/tootsuite/mastodon/blob/v2.1.0/.env.production.sample#L17)).

But in Mastodon v2.1.0 we do not work with `LOCAL_HTTPS=false`. You must set `LOCAL_HTTPS=true`.
